suricata/update/main.py

```diff
@@ -424,14 +424,25 @@ def manage_classification(suriconf, files):
 def handle_dataset_files(rule, dep_files):
     if not rule.enabled:
         return
-    load_attr = [el.strip() for el in rule.dataset.split(",") if "load" in el][0]
-    dataset_fname = os.path.basename(load_attr.split(" ")[1])
+    load_attr = [el.strip() for el in rule.dataset.split(",") if "load" in el]
+    state_attr = [el.strip() for el in rule.dataset.split(",") if "state" in el]
```

Comment:
"save" is another option, should it be handled here?

Reply:
No, this is about extracting a dataset file included in a rule set.

Reply:
Probably not since
- the file mentioned with `save` does not already exist in a freshly fetched ruleset
- as a result, nothing needs to be moved along with the rules